README.md

```diff
@@ -9,7 +9,7 @@
 
 # Visit [yomitan.wiki](https://yomitan.wiki) to learn more!
 
-:wave: **Yomitan is [the successor](https://foosoft.net/posts/passing-the-torch-to-yomitan/) to Yomichan** ([migration guide](https://yomitan.wiki/yomichan-migration/) which was [sunset](https://foosoft.net/posts/sunsetting-the-yomichan-project/) by its owner on Feb 26, 2023. We have made a number of foundational changes to ensure **the project stays alive, works on latest browser versions, and is easy to contribute to**.
+:wave: **Yomitan is [the successor](https://foosoft.net/posts/passing-the-torch-to-yomitan/) to Yomichan** [migration guide](https://yomitan.wiki/yomichan-migration/) which was [sunset](https://foosoft.net/posts/sunsetting-the-yomichan-project/) by its owner on Feb 26, 2023. We have made a number of foundational changes to ensure **the project stays alive, works on latest browser versions, and is easy to contribute to**.
```

Comment:
Hey actually the fix is to complete the parentheses after "migration guide" rather than remove the parentheses. Can you fix this?

Reply:
The sentence reads wrong if "migration guide" is not in a parentheses

Reply:
So should I place the parenthesis after the migration guide and link

Reply:
Yes notice how the sentence doesn't read well if the parentheses is removed
<img width="840" alt="image" src="https://github.com/user-attachments/assets/efef28a3-e7d7-4f4d-aca1-499c282825c4">

It should be:

> Yomitan is the successor to Yomichan (migration guide) which was sunset by its owner on Feb 26, 2023. We have made a number of foundational changes to ensure the project stays alive, works on latest browser versions, and is easy to contribute to.

Reply:
Okey